k8s.jsonnet

```diff
@@ -201,7 +210,7 @@
       apiVersion: "networking.k8s.io/v1beta1",
       metadata: {
         name: "wubloader",
-        labels: {app: "wubloader"},
+        labels: {app: "wubloader"} + (if (std.objectHas($.config, "ingress_labels")) then $.config.ingress_labels else {}),
```

Comment:
instead of checking for existence then defaulting to `{}`, use it here unconditionally then define it as `{}` instead of leaving it commented out (both work, but this is more consistent with our other config vars)